packages/analytics-js/src/components/userSessionManager/UserSessionManager.ts

```diff
@@ -142,9 +142,11 @@ class UserSessionManager implements IUserSessionManager {
       };
     }
```

Comment:
To simplify the logic further, we should add an else case here and assign `sessionInfo` to the default value.

In the next step, simply assign `sessionInfo` to the state.

Reply:
It'll avoid calling `isPersistenceEnabledForStorage` twice.

Reply:
Refactored